examples/shell/shell_common/cmd_server.cpp

```diff
@@ -0,0 +1,206 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <inttypes.h>
+#include <lib/core/CHIPCore.h>
+#include <lib/shell/Commands.h>
+#include <lib/shell/Engine.h>
+#include <lib/shell/commands/Help.h>
+#include <lib/support/CHIPArgParser.hpp>
+#include <lib/support/CHIPMem.h>
+#include <lib/support/CodeUtils.h>
+
+#include <app/server/Server.h>
+#include <app/util/af.h>
+#include <app/util/attribute-storage.h>
+#include <credentials/examples/DeviceAttestationCredsExample.h>
+
+using namespace chip;
+using namespace chip::Shell;
+using namespace chip::Credentials;
+using namespace chip::ArgParser;
+
+bool lowPowerClusterSleep()
+{
+    return true;
+}
+
+static chip::Shell::Engine sShellServerSubcommands;
```

Comment:
consider anonymous namespace instead of using 'static' (feels more C++ like), can place all variables and functions in there without repeating static everywhere.